src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1476,6 +1476,22 @@ protected override void HandleStackTraceRequestAsync(IRequestResponder<StackTrac
                         });
                     }
 
+                    if (!IsClientVS)
+                    {
+                        Parallel.ForEach(response.StackFrames, frame =>
+                        {
+                            if (frame.Source == null)
+                                return;
+                            var task = new Task<bool>(() => File.Exists(frame.Source.Path));
```

Comment:
if we do keep this, `Task.Run(...)` is preferrable to `new Task(...)` + `Task.Start()`